arborist/server.go

```diff
@@ -30,6 +30,11 @@ type Server struct {
 	stmts  *CachedStmts
 }
 
+type RequestPolicy struct {
```

Comment:
nice, let's use it in `handleUserGrantPolicy`, `handleClientGrantPolicy` and `handleGroupGrantPolicy`

Reply:
Im not sure how it works so the struct is
```
type RequestPolicy struct {
	PolicyName string `json:"policy"`
	ExpiresAt  string `json:"expires_at"`
}
```
But for `handleGroupGrantPolicy` its 
```
struct {
		PolicyName string `json:"policy"`
	}
```
I guess I could just try it out

Reply:
oh woops, you're right, it only works for `handleUserGrantPolicy`